tensorflow-examples/src/main/java/org/tensorflow/model/examples/mnist/training/LinearRegressionExample.java

```diff
@@ -0,0 +1,154 @@
+/*
+ *  Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *  =======================================================================
+ */
+package org.tensorflow.model.examples.mnist.training;
+
+import org.tensorflow.Graph;
+import org.tensorflow.Session;
+import org.tensorflow.Tensor;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.*;
+import org.tensorflow.op.math.*;
+import org.tensorflow.op.train.ApplyGradientDescent;
+import org.tensorflow.tools.Shape;
+import org.tensorflow.types.TFloat32;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+
+/**
+ * In this example the TensorFlow finds the weight and bias of the linear regression during 1 epoch,
+ * training on observations one by one.
+ *
+ * Also, the weight and bias are extracted and printed.
+ */
+public class LinearRegressionExample {
+    /**
+     * Amount of data points.
+     */
+    private static int N = 10;
+
+    /**
+     * This value is used to fill the Y placeholder in prediction.
+     */
+    private static final float NO_MEANING_VALUE_TO_PUT_IN_PLACEHOLDER = 2000f;
+
+    public static void main(String[] args) {
+        // Prepare the data
+        float[] xValues = {1f, 2f, 3f, 4f, 5f, 6f, 7f, 8f, 9f, 10f};
+        float[] yValues = new float[10];
+
+        Random rnd = new Random(42);
+
+        for (int i = 0; i < yValues.length; i++) {
+            yValues[i] = (float) (10 * xValues[i] + 2 + 0.1 * (rnd.nextDouble() - 0.5));
+        }
+
+        try (Graph graph = new Graph()) {
+            Ops tf = Ops.create(graph);
+
+            // Define placeholders
+            Placeholder<TFloat32> X = tf.placeholder(TFloat32.DTYPE, Placeholder.shape(Shape.scalar()));
+            Placeholder<TFloat32> Y = tf.placeholder(TFloat32.DTYPE, Placeholder.shape(Shape.scalar()));
```

Comment:
Normal variables should be name with lowercase in Java, so `x` and `y` would be a better fit

Reply:
done